data/lisbon.yaml

```diff
@@ -0,0 +1,35 @@
+---
+info:
+  city: Lisbon
+  link: lisbon
+  date: 12/17/2019
+  hour: '19:00'
+  icon: icons/lisbon/lisbon.svg
+  iconHover: icons/lisbon/lisbon-hover.svg
+  hostIcon: icons/paris/dashlane.svg
+  hostName: Dashlane
+  maxCapacity: 25
+site:
+  city: Paris
+  location: Dashlane
+  organizers:
+  googleMapsLink: https://goo.gl/maps/M8HCf3JqZ787HUzi8
+  calendarLink: https://www.google.com/calendar/render?action=TEMPLATE&text=QueerJS+Lisbon&location=Dashlane+R.+Garrett+12+2%C2%BA+piso%2C+1200-219+Lisboa%2C+Portugal&dates=20191217T180000Z%2F20191217T210000Z
+mainOrganizer:
+  - name: Paola Ducolin
```

Comment:
Not to nag, but it'd be great to add you to the organizers page for both Paris and now Lisbon so that people know who to contact (especially for Code of Conduct related issues). Can you either add yourself or reply to my message on discord?

Reply:
I did this :)

Reply:
@carolstran can't see your message on Discord, I just loggedin as Paola

Reply:
@pducolin Sara did it! No worries 😁